arches/app/utils/thumbnail_factory.py

```diff
@@ -39,4 +39,11 @@ def create(self):
             logger.info(f"A thumbnail generator isn't specified.  Set 'THUMBNAIL_GENERATOR' in settings.py to enable generation of thumbnails for uploded files.")
             return None
 
-ThumbnailGeneratorInstance = ThumbnailFactory().create()
+
+# system_settings does queries (in update_from_db), but queries are
+# not allowed at the module level, so guard under apps.ready.
+# https://github.com/django/django/blob/adae619426b6f50046b3daaa744db52989c9d6db/docs/topics/testing/overview.txt#L201
+if apps.ready:
+    ThumbnailGeneratorInstance = ThumbnailFactory().create()
+else:
+    ThumbnailGeneratorInstance = object()
```

Comment:
This needs more testing to make sure I didn't break the functionality, but as linked in the docs, there could be subtle breakages from having module-level queries anyway, so now's a good time to do something about it.

Reply:
I ended up debugging this because django.setup() imports arches in a particular way.

Reply:
Never mind, I just worked around this by hiding the circular import of this module.

I think the "module-time import" is not an issue because `THUMBNAIL_GENERATOR` is defined in core arches and so we never go to the database to get the setting. Resolving.